regression/tests/whitelabel/course_enrollment_test.py

```diff
@@ -10,17 +10,14 @@
     LmsApiClient,
     EnrollmentApiClient
 )
-
 from regression.pages.whitelabel.basket_page import (
     BasketPage, CyberSourcePage, MultiSeatBasketPage, SingleSeatBasketPage
 )
 from regression.pages.whitelabel.receipt_page import ReceiptPage
 from regression.pages.whitelabel.courses_page import CoursesPage
 from regression.pages.whitelabel.course_about_page import CourseAboutPage
-
 from regression.pages.whitelabel.const import (
     PASSWORD,
-    TIME_OUT_LIMIT,
```

Comment:
oh good catch. I think you can remove it from regression/pages/whitelabel/const.py also